typescript/cli/src/config/utils.ts

```diff
@@ -0,0 +1,18 @@
+import { HookConfig, HookType, IsmConfig, IsmType } from '@hyperlane-xyz/sdk';
+
+import { logGray } from '../logger.js';
+
+export function callWithConfigCreationLogs<T extends IsmConfig | HookConfig>(
```

Comment:
are there other benefits of having this function other than the try-catch and log? additional complexity might not worth the tradeoff for a bit of dry-ness imo